config/kapp-rebase-rules.yml

```diff
@@ -1,7 +1,9 @@
 ---
 apiVersion: kapp.k14s.io/v1alpha1
 kind: Config
-
+metadata:
+  name: "test"
```

Comment:
maybe call this kapp-config? We'll also need an entry in the new file: https://github.com/cloudfoundry/cf-for-k8s/blob/master/config/0-min-kapp-version.yml